src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -31,6 +31,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <None Remove="**/*$(DefaultLanguageSourceExtension)" Condition=" '$(EnableDefaultCompileItems)' == 'true' "/>
     <None Remove="**/*.resx" Condition=" '$(EnableDefaultEmbeddedResourceItems)' == 'true' "/>
   </ItemGroup>
+  <ItemGroup Condition=" '$(EnableDefaultDependentUponMetadata)' == 'true' And '$(EnableDefaultResxDependentUponMetadata)' == 'true' ">
```

Comment:
Have you measured the perf hit to evaluation? This reminds me of the automatic Link metadata that ended up being a perf problem.

Reply:
Not yet.